Makefile.am

```diff
@@ -96,3 +96,8 @@ include $(top_srcdir)/makefiles/test-cases.mak
 
 include $(top_srcdir)/makefiles/help.mak
 
+if HAVE_RST2MAN
+%.1.in: %.1.in.rst
```

Comment:
This might not be portable to non-GNU make. I'm not sure of the exact portable syntax for suffix rules with Automake and these complicated extensions, but I think it's something like:

```
SUFFIXES = .1.in .1.in.rst
.1.in.rst.1.in:
    rst2man $< $@
```

Also it might cause a weird dependency problem if this rule produces files that Autoconf substitutes in, where you need to run `make` once first to produce the Autoconf input file and then run `configure` again to produce the final output, or it even might not work at all if you tried with a clean source tree. It's also entirely possible I just misunderstood what's happening here :)